server/src/main/java/org/apache/iotdb/db/mpp/execution/config/AuthorizerConfigTask.java

```diff
@@ -19,38 +19,32 @@
 
 package org.apache.iotdb.db.mpp.execution.config;
 
-import org.apache.iotdb.common.rpc.thrift.TSStatus;
-import org.apache.iotdb.commons.exception.BadNodeUrlException;
 import org.apache.iotdb.confignode.rpc.thrift.TAuthorizerReq;
 import org.apache.iotdb.db.auth.AuthException;
-import org.apache.iotdb.db.client.ConfigNodeClient;
-import org.apache.iotdb.db.mpp.sql.planner.plan.node.metedata.write.AuthorNode;
+import org.apache.iotdb.db.auth.authorizer.AuthorizerManager;
+import org.apache.iotdb.db.mpp.sql.analyze.QueryType;
 import org.apache.iotdb.db.mpp.sql.statement.sys.AuthorStatement;
-import org.apache.iotdb.rpc.IoTDBConnectionException;
-import org.apache.iotdb.rpc.StatementExecutionException;
-import org.apache.iotdb.rpc.TSStatusCode;
+import org.apache.iotdb.db.qp.physical.sys.AuthorPlan;
 
 import com.google.common.util.concurrent.ListenableFuture;
 import com.google.common.util.concurrent.SettableFuture;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.util.Locale;
-
 public class AuthorizerConfigTask implements IConfigTask {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(AuthorizerConfigTask.class);
 
   private AuthorStatement authorStatement;
+  private AuthorizerManager authorizerManager = AuthorizerManager.getInstance();
```

Comment:
Is using a ClusterAuthorizer enough?

Reply:
fix

Reply:
if use ClusterAuthorizer ， The permission class cannot be reflected from the configuration.

Reply:
ok

Reply:
> if use ClusterAuthorizer ， The permission class cannot be reflected from the configuration.

fix